front-end/package.json

```diff
@@ -4,14 +4,32 @@
   "private": true,
   "proxy": "http://backend:5000",
   "dependencies": {
+    "bootstrap": "^3.3.7",
+    "lodash": "^4.17.11",
+    "moment": "^2.22.2",
+    "node-sass-chokidar": "^1.3.3",
+    "npm-run-all": "^4.1.3",
```

Comment:
what is this?

Reply:
`you may find it convenient to run watch-css automatically with npm start, and run build-css as a part of npm run build. You can use the && operator to execute two scripts sequentially. However, there is no cross-platform way to run two scripts in parallel, so we will install a package for this: npm-run-all` - Taken directly from create-react-app readme.